services/github/github-commits-since.service.js

```diff
@@ -77,6 +81,52 @@ module.exports = class GithubCommitsSince extends GithubAuthV3Service {
         }),
         documentation,
       },
+      {
+        title: 'GitHub commits since latest including pre-releases',
```

Comment:
This is the example for latest by date right? If so then it'd be good to mention that explicitly in the title to ensure the difference is clear for end users. Check out the example titles used in the GH Release service for a reference

Reply:
I copy+pasted from GH Release into here and then edited them. The pure latest (no pre-releases) is above this, so I left this with the same wording. Just changed them all to be explicit about "by date", though they're not explicitly using the `sort=date` query param, just that it's the default.